cmd/geth/blsaccountcmd.go

```diff
@@ -0,0 +1,549 @@
+package main
+
+import (
+	"context"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/google/uuid"
+	"github.com/logrusorgru/aurora"
+	"github.com/prysmaticlabs/prysm/crypto/bls"
+	"github.com/prysmaticlabs/prysm/encoding/bytesutil"
+	"github.com/prysmaticlabs/prysm/io/file"
+	"github.com/prysmaticlabs/prysm/io/prompt"
+	"github.com/prysmaticlabs/prysm/validator/accounts"
+	"github.com/prysmaticlabs/prysm/validator/accounts/iface"
+	"github.com/prysmaticlabs/prysm/validator/accounts/petnames"
+	"github.com/prysmaticlabs/prysm/validator/accounts/wallet"
+	"github.com/prysmaticlabs/prysm/validator/keymanager"
+	"github.com/prysmaticlabs/prysm/validator/keymanager/imported"
+	keystorev4 "github.com/wealdtech/go-eth2-wallet-encryptor-keystorev4"
+	"gopkg.in/urfave/cli.v1"
+
+	"github.com/ethereum/go-ethereum/cmd/utils"
+)
+
+const (
+	BLSWalletPath   = "bls/wallet"
+	BLSKeystorePath = "bls/keystore"
+)
+
+var (
+	au = aurora.NewAurora(true)
+)
+
+var (
+	blsCommand = cli.Command{
+		Name:      "bls",
+		Usage:     "Manage BLS wallet and accounts",
+		ArgsUsage: "",
+		Category:  "BLS ACCOUNT COMMANDS",
+		Description: `
+
+Manage BLS wallet and accounts, before creating or importing BLS accounts, create
+BLS wallet first. One BLS wallet is enough for all BLS accounts, the first BLS
+account in the wallet will be used to vote for fast finality now.
+
+It only supports interactive mode now, when you are prompted for password, please
+input your password, and take care the wallet password is different from accounts'
```

Comment:
`the wallet password is different from` -> `the wallet password which is different from`

Reply:
fixed